jest.config.ts

```diff
@@ -5,6 +5,6 @@ export default async (): Promise<Config> => {
     transform: { "^.+\\.ts?$": "ts-jest" },
     testEnvironment: "node",
     testRegex: "/tests/.*\\.(test|spec)?\\.(ts|tsx)$",
-    moduleFileExtensions: ["ts", "json", "node"],
+    moduleFileExtensions: ["js", "ts", "json", "node"],
```

Comment:
What do we need this? I don't see any js files here.

Reply:
Undoing this makes `yarn test` fail (it seems to require at least .js in this list)